### PR TITLE
:sparkles: Add PDF fixer

### PR DESCRIPTION
**Pull request by alexdesousa**
This PR adds two options for fixing:

- [x] Broken PDFs.
- [x] PDFs without embedded fonts.

We can call  `puppeteer-pdf` as:

```bash
$ ./puppeteer-pdf.js --brokenPdf "/path/to/my/broken/file.pdf" --fixedPdf "/path/to/fixed/file.pdf"
```

> **There's a known drawback:** The resulting file will have only one page, so it's better to split PDFs in several pages, fixed every single one of them and finally merge them

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -10,27 +10,53 @@ import { prepareOptions } from "./src/options";
...
+    `
+
+    await page.setContent(html);
+    await page.waitForSelector('embed[type="application/pdf"]');
+
+    await page.pdf({path: options.fixedPdf, printBackground: true});
+
+    await browser.close();
 
-  await page.pdf(options);
```
Comment (pdomaleczny) [inline main.js]: I would do it this way so it is easier to read :D 

```
import fixBrokenPdf from "./fixBrokenPdf";

if (!options?.brokenPdf && !options?.fixedPdf) {
  fixBrokenPDF(options)
  return;
}
.... rest of working code below
```
In `fixBrokenPdf.js`:
```
const fixBrokenPdf = (options) => { ... }
export default fiBrokenPdf;
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contractbook/puppeteer-pdf",
   "author": "Patryk Domałeczny <patryk.domaleczny@gmail.com>",
-  "version": "1.4.0",
+  "version": "1.6.0",
```
Comment (pdomaleczny) [inline package.json:4]: We need to remember to push the new tag version and update that in FS images :D

File: main.js (hunk 1 above)
Comment (alexdesousa) [reply main.js]: That's way better :100: I didn't know how to import properly xD

File: package.json (hunk 2 above)
Comment (alexdesousa) [reply package.json:4]: Yep, I'm in progress with this one :) Once this is in master, I'll update the tag.

**Review by pdomaleczny — APPROVED** [review]

**Pull request merged**